gui.py

```diff
@@ -145,6 +162,9 @@ def configureEnvironment(self):
 
     def configureEnvironmentColorNames(self):
         return ["Pollution"]
+
+    def configureGraphNames(self):
+        return ["Sugar Histogram", "Spice Histogram", "Tag Histogram", "Age Histogram", "Lorenz Curve"]
```

Comment:
Let's call `Lorenz Curve` the `Gini Coefficient` graph instead.

Please also alphabetize these and the social network graph options.

Reply:
Done, but I kept some backend references to `lorenzCurve` for clarity (the polygon tag, for example).